No problem, I'm right now writing some tests. But I'm running in the same issues as mentioned in #472. Most python tests just fail with the following message:

```
+ /home/panmari/.cache/bazel/_bazel_panmari/f5b407cdd255b7813aa5a26beb5d6822/tensorflow/bazel-out/local_linux-fastbuild/bin/tensorflow/python/gen_docs_test.runfiles/tensorflow/python/gen_docs_combined --out_dir /home/panmari/.cache/bazel/_bazel_panmari/f5b407cdd255b7813aa5a26beb5d6822/tensorflow/_tmp/gen_docs_test_2
RuntimeError: module compiled against API version a but this version of numpy is 9
Traceback (most recent call last):
  File "/home/panmari/.cache/bazel/_bazel_panmari/f5b407cdd255b7813aa5a26beb5d6822/tensorflow/bazel-out/local_linux-fastbuild/bin/tensorflow/python/gen_docs_test.runfiles/tensorflow/python/framework/gen_docs_combined.py", line 23, in <module>
    import tensorflow.python.platform
  File "/home/panmari/.cache/bazel/_bazel_panmari/f5b407cdd255b7813aa5a26beb5d6822/tensorflow/bazel-out/local_linux-fastbuild/bin/tensorflow/python/gen_docs_test.runfiles/tensorflow/__init__.py", line 23, in <module>
    from tensorflow.python import *
  File "/home/panmari/.cache/bazel/_bazel_panmari/f5b407cdd255b7813aa5a26beb5d6822/tensorflow/bazel-out/local_linux-fastbuild/bin/tensorflow/python/gen_docs_test.runfiles/tensorflow/python/__init__.py", line 50, in <module>
    from tensorflow.python.framework.framework_lib import *
  File "/home/panmari/.cache/bazel/_bazel_panmari/f5b407cdd255b7813aa5a26beb5d6822/tensorflow/bazel-out/local_linux-fastbuild/bin/tensorflow/python/gen_docs_test.runfiles/tensorflow/python/framework/framework_lib.py", line 62, in <module>
    from tensorflow.python.framework.ops import Graph
  File "/home/panmari/.cache/bazel/_bazel_panmari/f5b407cdd255b7813aa5a26beb5d6822/tensorflow/bazel-out/local_linux-fastbuild/bin/tensorflow/python/gen_docs_test.runfiles/tensorflow/python/framework/ops.py", line 40, in <module>
    from tensorflow.python.framework import versions
  File "/home/panmari/.cache/bazel/_bazel_panmari/f5b407cdd255b7813aa5a26beb5d6822/tensorflow/bazel-out/local_linux-fastbuild/bin/tensorflow/python/gen_docs_test.runfiles/tensorflow/python/framework/versions.py", line 24, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "/home/panmari/.cache/bazel/_bazel_panmari/f5b407cdd255b7813aa5a26beb5d6822/tensorflow/bazel-out/local_linux-fastbuild/bin/tensorflow/python/gen_docs_test.runfiles/tensorflow/python/pywrap_tensorflow.py", line 28, in <module>
    _pywrap_tensorflow = swig_import_helper()
  File "/home/panmari/.cache/bazel/_bazel_panmari/f5b407cdd255b7813aa5a26beb5d6822/tensorflow/bazel-out/local_linux-fastbuild/bin/tensorflow/python/gen_docs_test.runfiles/tensorflow/python/pywrap_tensorflow.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow', fp, pathname, description)
ImportError: numpy.core.multiarray failed to import
```

--- a/tensorflow/python/ops/image_ops_test.py
+++ b/tensorflow/python/ops/image_ops_test.py
@@ -564,49 +564,54 @@ class ResizeImagesTest(test_util.TensorFlowTestCase):
              image_ops.ResizeMethod.BICUBIC,
              image_ops.ResizeMethod.AREA]
 
+  TYPES = [np.uint8, np.int8, np.int16, np.int32, np.int64,
+           np.float, np.double]
+
   def testNoOp(self):
     img_shape = [1, 6, 4, 1]
     single_shape = [6, 4, 1]
-    data = [128, 128, 64, 64,
-            128, 128, 64, 64,
-            64, 64, 128, 128,
-            64, 64, 128, 128,
+    # This test is also conducted with int8, so 127 is the maximum value that can be used.
+    data = [127, 127, 64, 64,
+            127, 127, 64, 64,
+            64, 64, 127, 127,
+            64, 64, 127, 127,
             50, 50, 100, 100,
             50, 50, 100, 100]
-    img_np = np.array(data, dtype=np.uint8).reshape(img_shape)
-
     target_height = 6
     target_width = 4
 
-    for opt in self.OPTIONS:
-      with self.test_session() as sess:
-        image = constant_op.constant(img_np, shape=img_shape)
-        y = image_ops.resize_images(image, target_height, target_width, opt)
-        yshape = array_ops.shape(y)
-        resized, newshape = sess.run([y, yshape])
-        self.assertAllEqual(img_shape, newshape)
-        self.assertAllClose(resized, img_np, atol=1e-5)
+    for type in self.TYPES:
+      img_np = np.array(data, dtype=type).reshape(img_shape)
 
-    # Resizing with a single image must leave the shape unchanged also.
-    with self.test_session():
-      img_single = img_np.reshape(single_shape)
-      image = constant_op.constant(img_single, shape=single_shape)
-      y = image_ops.resize_images(image, target_height, target_width,
-                                  self.OPTIONS[0])
-      yshape = array_ops.shape(y)
-      newshape = yshape.eval()
-      self.assertAllEqual(single_shape, newshape)
+      for opt in self.OPTIONS:
+        with self.test_session() as sess:
+          image = constant_op.constant(img_np, shape=img_shape)
+          y = image_ops.resize_images(image, target_height, target_width, opt)
+          yshape = array_ops.shape(y)
+          resized, newshape = sess.run([y, yshape])
+          self.assertAllEqual(img_shape, newshape)
+          self.assertAllClose(resized, img_np, atol=1e-5)
+
+      # Resizing with a single image must leave the shape unchanged also.
+      with self.test_session():
+        img_single = img_np.reshape(single_shape)
+        image = constant_op.constant(img_single, shape=single_shape)
+        y = image_ops.resize_images(image, target_height, target_width,
+                                    self.OPTIONS[0])
+        yshape = array_ops.shape(y)
+        newshape = yshape.eval()
+        self.assertAllEqual(single_shape, newshape)
 
   def testResizeDown(self):
-
-    data = [128, 128, 64, 64,
-            128, 128, 64, 64,
-            64, 64, 128, 128,
-            64, 64, 128, 128,
+    # This test is also conducted with int8, so 127 is the maximum value that can be used.
+    data = [127, 127, 64, 64,
+            127, 127, 64, 64,
+            64, 64, 127, 127,
+            64, 64, 127, 127,
             50, 50, 100, 100,
             50, 50, 100, 100]
-    expected_data = [128, 64,
-                     64, 128,
+    expected_data = [127, 64,
+                     64, 127,
                      50, 100]
     target_height = 3
     target_width = 2
@@ -617,59 +622,61 @@ class ResizeImagesTest(test_util.TensorFlowTestCase):
                      [target_height, target_width, 1]]
 
     for target_shape, img_shape in zip(target_shapes, img_shapes):
-      img_np = np.array(data, dtype=np.uint8).reshape(img_shape)
 
-      for opt in self.OPTIONS:
-        with self.test_session():
-          image = constant_op.constant(img_np, shape=img_shape)
-          y = image_ops.resize_images(image, target_height, target_width, opt)
-          expected = np.array(expected_data).reshape(target_shape)
-          resized = y.eval()
-          self.assertAllClose(resized, expected, atol=1e-5)
+      for type in self.TYPES:
+        img_np = np.array(data, dtype=type).reshape(img_shape)
+
+        for opt in self.OPTIONS:
+          with self.test_session():
+            image = constant_op.constant(img_np, shape=img_shape)
+            y = image_ops.resize_images(image, target_height, target_width, opt)
+            expected = np.array(expected_data).reshape(target_shape)
+            resized = y.eval()
+            self.assertAllClose(resized, expected, atol=1e-5)
 
   def testResizeUp(self):
     img_shape = [1, 3, 2, 1]
-    data = [128, 64,
-            64, 128,
+    data = [64, 32,
+            32, 64,
             50, 100]
-    img_np = np.array(data, dtype=np.uint8).reshape(img_shape)
-
     target_height = 6
     target_width = 4
     expected_data = {}
     expected_data[image_ops.ResizeMethod.BILINEAR] = [
-        128.0, 96.0, 64.0, 64.0,
-        96.0, 96.0, 96.0, 96.0,
-        64.0, 96.0, 128.0, 128.0,
-        57.0, 85.5, 114.0, 114.0,
+        64.0, 48.0, 32.0, 32.0,
+        48.0, 48.0, 48.0, 48.0,
+        32.0, 48.0, 64.0, 64.0,
+        41.0, 61.5, 82.0, 82.0,
         50.0, 75.0, 100.0, 100.0,
         50.0, 75.0, 100.0, 100.0]
     expected_data[image_ops.ResizeMethod.NEAREST_NEIGHBOR] = [
-        128.0, 128.0, 64.0, 64.0,
-        128.0, 128.0, 64.0, 64.0,
-        64.0, 64.0, 128.0, 128.0,
-        64.0, 64.0, 128.0, 128.0,
+        64.0, 64.0, 32.0, 32.0,
+        64.0, 64.0, 32.0, 32.0,
+        32.0, 32.0, 64.0, 64.0,
+        32.0, 32.0, 64.0, 64.0,
         50.0, 50.0, 100.0, 100.0,
         50.0, 50.0, 100.0, 100.0]
     expected_data[image_ops.ResizeMethod.AREA] = [
-        128.0, 128.0, 64.0, 64.0,
-        128.0, 128.0, 64.0, 64.0,
-        64.0, 64.0, 128.0, 128.0,
-        64.0, 64.0, 128.0, 128.0,
+        64.0, 64.0, 32.0, 32.0,
+        64.0, 64.0, 32.0, 32.0,
+        32.0, 32.0, 64.0, 64.0,
+        32.0, 32.0, 64.0, 64.0,
         50.0, 50.0, 100.0, 100.0,
         50.0, 50.0, 100.0, 100.0]
 
-    for opt in [
-        image_ops.ResizeMethod.BILINEAR,
-        image_ops.ResizeMethod.NEAREST_NEIGHBOR,
-        image_ops.ResizeMethod.AREA]:
-      with self.test_session():
-        image = constant_op.constant(img_np, shape=img_shape)
-        y = image_ops.resize_images(image, target_height, target_width, opt)
-        resized = y.eval()
-        expected = np.array(expected_data[opt]).reshape(
-            [1, target_height, target_width, 1])
-        self.assertAllClose(resized, expected, atol=1e-05)
+    for type in self.TYPES:
+      for opt in [
+          image_ops.ResizeMethod.BILINEAR,
+          image_ops.ResizeMethod.NEAREST_NEIGHBOR,
+          image_ops.ResizeMethod.AREA]:
+        with self.test_session():
+          img_np = np.array(data, dtype=type).reshape(img_shape)
+          image = constant_op.constant(img_np, shape=img_shape)
+          y = image_ops.resize_images(image, target_height, target_width, opt)
+          resized = y.eval()
+          expected = np.array(expected_data[opt]).reshape(
+              [1, target_height, target_width, 1])
+          self.assertAllClose(resized, expected, atol=1e-05)
 
   def testResizeUpBicubic(self):
     img_shape = [1, 6, 6, 1]

